endpoints/openrtb2/amp_auction_test.go

```diff
@@ -129,7 +228,7 @@ func TestAMPPageInfo(t *testing.T) {
 }
 
 func TestGDPRConsent(t *testing.T) {
-	consent := "BOu5On0Ou5On0ADACHENAO7pqzAAppY"
+	consent := "CPdiPIJPdiPIJACABBENAzCv_____3___wAAAQNd_X9cAAAAAAAA"
```

Comment:
Why did you need to change this test consent string?

Reply:
If I recall my consent strings, the old is TCF1 and the new is TCF2

Reply:
After the logic change the consent string that was removed was no longer passing validation. I believe @hhhjort s right, it must be a TCF1 string.